website/docs/cloud-docs/workspaces/variables/index.mdx

```diff
@@ -99,9 +99,15 @@ For example, if you apply `A_Variable_Set` and `B_Variable_Set` to the same work
 
 Non-global variable sets always take precedence over global variable sets that are applied to all workspaces within an organization. Terraform does not allow global variable sets to contain variables with the same key, so they cannot conflict.
 
-### 6. `auto.tfvars` Variable Files
+### 6. Variable Definitions Files
 
-Variables in the Terraform Cloud workspace and variables provided through the command line always overwrite variables with the same key from files ending in `auto.tfvars`.
+Variables in the Terraform Cloud workspace and variables provided through the command line always overwrite variables with the same key from files ending in `*.tfvars`.
```

Comment:
Only files named exactly terraform.tfvars or ending with .auto.tfvars are automatically loaded. If your config contains foo.tfvars, it's completely ignored (unless you run from the CLI `terraform apply -var-file=foo.tfvars`)